public/app/features/alerting/unified/utils/datasource.ts

```diff
@@ -51,12 +51,22 @@ export function getRulesDataSource(rulesSourceName: string) {
 
 export function getAlertManagerDataSources() {
   return getAllDataSources()
-    .filter(
-      (ds): ds is DataSourceInstanceSettings<AlertManagerDataSourceJsonData> => ds.type === DataSourceType.Alertmanager
-    )
+    .filter(isAlertmanagerDataSourceInstance)
     .sort((a, b) => a.name.localeCompare(b.name));
 }
 
+export function isAlertmanagerDataSourceInstance(
```

Comment:
These functions will help with type narrowing to `AlertManagerDataSourceJsonData` so the type system knows when we can access properties on `jsonData` that are unique to Alertmanager data sources